content/applications/sales/crm/optimize/lead_enrichment.rst

```diff
@@ -0,0 +1,123 @@
+===============
+Lead enrichment
+===============
+
+Lead enrichment is a service that provides business information for a contact attached to a lead.
```

Comment:
```suggestion
*Lead enrichment* is a service that provides business information for a contact attached to a lead.
```
I'm suggesting italics here because it is, essentially, defining what 'lead enrichment' is...